D6StarWars/D6StarWars.html

```diff
@@ -983,26 +985,26 @@ <h4>Force Powers</h4>
       <label>Ship Weapons</label>
       <fieldset class="repeating_shipweapon">
         <label class="bold">Weapon Name<input name="attr_weapon_name" type="text"></label>
-        <div style="margin-bottom:10px;">
-          <span>Scale:<input class="smallnumber" name="attr_weapon_scale" type="text" value=""></span>
+        <div>
+          <span>Scale:<input class="40px" name="attr_weapon_scale" type="text" value=""></span>
           <span>Fire Arc:<input name="attr_weapon_arc" type="text" value=""></span>
           <span>Crew:<input name="attr_weapon_crew" type="text" value=""></span>
         </div>
 
-        <div style="margin-bottom:10px;">
+        <div>
           <span>Skill:<input name="attr_weapon_skill" type="text" value=""></span>
-          <span>Fire Control:<input class="smallnumber" name="attr_weapon_fire_control" type="number" value=""> D</span>
-          <span style="margin-left: 15px;">Space Range:<input class="mediumnumber" name="attr_weapon_range_space" type="number"
+          <span>Fire Control:<input class="40px" name="attr_weapon_fire_control" type="number" value=""> D</span>
+          <span class="range">Space Range:<input class="mediumnumber" name="attr_weapon_range_space" type="number"
                                                               value=""></span>
-          <span>Atmosphere Range:<input class="smallnumber" name="attr_weapon_range_atmosphere" type="number" value=""></span>
+          <span>Atmosphere Range:<input class="40px" name="attr_weapon_range_atmosphere" type="number" value=""></span>
         </div>
 
-        <div style="margin-bottom:10px;">
-          <span>Damage:<input class="smallnumber" name="attr_weapon_damage" type="number" value="3">D+<input
-              class="smallnumber" name="attr_weapon_damage_pip" type="number" value="0"> <button class="d6-dice"
-                                                                                                 name="roll_ship_wpn_dmg"
-                                                                                                 type="roll"
-                                                                                                 value="&{template:swd6} {{name=Weapon: @{weapon_name} }} {{subtitle=Ship: @{ship_name} }} {{Roll=[[(@{weapon_damage}- 1)d6cf0cs0 + @{weapon_damage_pip} +(1d6@{wilddie})]] damage}}">DMG</button></span>
+        <div>
+          <span>Damage:
+            <input class="40px" name="attr_weapon_damage" type="number" value="3">D+<input
+              class="smallnumber" name="attr_weapon_damage_pip" type="number" value="0">
```

Comment:
Hee hoo more value setting! This means that the value is set for **EVERY** sheet!